<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 250, 250);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_appSetup">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/btn_setup.png</normaloff>:/images/btn_setup.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_courseId">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>258</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="placeholderText">
     <string>请输入课堂编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>课堂编号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_createClass">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>445</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #28b28b;/*设置边框的粗细，以及颜色*/
color: rgb(40, 178, 139);
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
    </property>
    <property name="text">
     <string>创建教室</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_joinClass">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>445</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #cccccc;/*设置边框的粗细，以及颜色*/
color: rgb(255, 255, 255);
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
background-color: #cccccc;</string>
    </property>
    <property name="text">
     <string>进入教室</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_userId">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>用户ID:12345656</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(40, 178, 139);</string>
    </property>
    <property name="text">
     <string>昵称:无名1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_role_teacher">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>老师</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_role_student">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>学生</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_courseStatus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="logo_index">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_coursePwd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>375</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 8px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>请输入进入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_coursePwd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>进入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_courseName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>510</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;
background-color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string>历史课程</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>516</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/img_jump.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_share">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>复制课堂编号</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(250, 250, 250, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/btn_share.png</normaloff>:/images/btn_share.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_debug">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab_appName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>165</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>启明课堂</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
